test/unit/plugins/rest_spec.js

```diff
@@ -678,20 +682,31 @@ describe("The Rest plugin", function () {
 					return instances.getInstance({ id : instance.id });
 				})
 				.then(function (instance) {
+					terminateInstancesStub = Sinon.stub(awsAdapter, "terminateInstances").returns(
+						Bluebird.resolve({
+							id       : instance.id,
+							ami      : instance.ami,
+							type     : instance.type,
+							state    : "terminated",
+							uri      : instance.uri,
+							revision : instance.revision + 2
+						})
+					);
+
 					updatedInstance = new Instance({
 						id       : instance.id,
 						ami      : instance.ami,
 						type     : instance.type,
-						state    : "failed",
+						state    : "terminated",
 						uri      : instance.uri,
-						revision : instance.revision + 1
+						revision : instance.revision + 2
```

Comment:
It shouldn't really matter since I'm stubbing `terminateInstances()`, but the reason I've done this is because when this is not stubbed the `state` is first set to `terminating` and when AWS says it's been successfully terminated it is set to `terminated`. Since `updateInstance()` is called twice the final revision would be the original + 2.